zeno-cli/zeno-cli.go

```diff
@@ -520,8 +525,8 @@ func printQueryStats(stderr io.Writer, md *common.QueryMetaData) {
 	fmt.Fprintf(stderr, "# As Of:      %v\n", md.AsOf.In(time.UTC).Format(time.RFC1123))
 	fmt.Fprintf(stderr, "# Until:      %v\n", md.Until.In(time.UTC).Format(time.RFC1123))
 	fmt.Fprintf(stderr, "# Resolution: %v\n", md.Resolution)
-	fmt.Fprintln(stderr, "\n")
+	fmt.Fprint(stderr, "\n")
```

Comment:
Drive-by. Compiler complains about the extra linefeed.

Reply:
I guess without the `ln` we need to add an extra line feed to keep the formatting, right?